lbrynet/conf.py

```diff
@@ -28,12 +28,10 @@
 POINTTRADER_SERVER = 'http://ec2-54-187-192-68.us-west-2.compute.amazonaws.com:2424'
 #POINTTRADER_SERVER = 'http://127.0.0.1:2424'
 
-if IS_DEVELOPMENT_VERSION:
-    SEARCH_SERVERS = ["http://107.170.207.64:50005"]
-else:
-    SEARCH_SERVERS = ["http://lighthouse1.lbry.io:50005",
-                      "http://lighthouse2.lbry.io:50005",
-                      "http://lighthouse3.lbry.io:50005"]
+
+SEARCH_SERVERS = ["http://lighthouse1.lbry.io:50005",
+                    "http://lighthouse2.lbry.io:50005",
+                    "http://lighthouse3.lbry.io:50005"]
```

Comment:
@jobevers @jackrobison possibly remove search logic from daemon entirely and put only in the interface?